governance/MEMBERSHIP_SURVEY.md

```diff
@@ -0,0 +1,32 @@
+
+# Membership Survey
+
+Surveys for current and past members to collect data on member wants and needs. following #375 issue.
+
+## Suggested Questions
+
+@amiller-gh
+
+What is your current happiness (perceived value?) of Ind. Membership program?
+Net promoter score – how likely would you recommend the program to a friend?
+How did you hear about the IM program?
+In your words, what value do you expect to receive from an open source individual membership?
+(For people with lapsed membership) What made you not renew?
+
+  
+
+@joesepi
+
+Why did you become an Individual Member? (If lapsed, why?)
```

Comment:
`(If lapsed, why?)` is a duplicate of the final question in @amiller-gh's section